torchrl/objectives/returns/functional.py

```diff
@@ -88,3 +93,47 @@ def td_advantage_estimate(
     not_done = 1 - done.to(next_state_value.dtype)
     advantage = reward + gamma * not_done * next_state_value - state_value
     return advantage
+
+
+def td_lambda_advantage_estimate(
+    gamma: float,
+    lamda: float,
+    state_value: torch.Tensor,
+    next_state_value: torch.Tensor,
+    reward: torch.Tensor,
+    done: torch.Tensor,
+) -> Tuple[torch.Tensor, torch.Tensor]:
+    """
+    Get generalized advantage estimate of a trajectory
+    Refer to "HIGH-DIMENSIONAL CONTINUOUS CONTROL USING GENERALIZED ADVANTAGE ESTIMATION"
+    https://arxiv.org/pdf/1506.02438.pdf for more context.
+
+    Args:
+        gamma (scalar): exponential mean discount.
```

Comment:
About the gamma and lambda, it can be different in different timesteps and batchs, should we consider that? It is considered in rlax, not common choice though.

Reply:
Yes good point!
Again I was planning to do that in a separate PR as it could be the case for GAE too.

Reply:
Great

Reply:
I did not follow the point about "it can be different in different timesteps and batchs, should we consider that". Could you please share some pointers